### PR TITLE
chore: Prepare v1.1.0 release

### PR DESCRIPTION
**Pull request by ashnamehrotra**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, using `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when the PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by pmengelbert — APPROVED** [review]
LGTM, let's get a review from @sozercan as well

**Review by sozercan — APPROVED** [review]
few minor comments, otherwise LGTM

File: docs/versions.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 [
+  "v1.1.0",
```
Comment (sozercan) [inline docs/versions.json]:
```suggestion
  "v1.1.x",
```

File: docs/versioned_docs/version-v1.1.0/installation.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+title: Installation
+---
+
+## Manifest
+
+To install Eraser with the manifest file, run the following command:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/Azure/eraser/v1.1.0-beta.0/deploy/eraser.yaml
```
Comment (sozercan) [inline docs/versioned_docs/version-v1.1.0/installation.md]:
```suggestion
kubectl apply -f https://raw.githubusercontent.com/Azure/eraser/v1.1.0/deploy/eraser.yaml
```

**Pull request merged**